pkg/test/resourcefixture/testdata/basic/containerattached/v1beta1/containerattachedclusterfull/_generated_object_containerattachedclusterfull.golden.yaml

```diff
@@ -52,4 +52,9 @@ status:
     reason: UpToDate
     status: "True"
     type: Ready
+  createTime: "1970-01-01T00:00:00Z"
+  kubernetesVersion: "1.28"
   observedGeneration: 2
+  state: RUNNING
+  uid: "12345678"
+  updateTime: "1970-01-01T00:00:00Z"
```

Comment:
We want to update the merge-in code to reflect this part, this is important for KCC to maintain the resource and align the expected behavior with the GCP service.